doc/faq.rst

```diff
@@ -101,8 +101,8 @@ frequency ranges, it is not truly '1/f' across all frequencies.
 From the physics perspective, '1/f' activity is sometimes referred to as 'noise',
 as shorthand for 'statistical noise' or `coloured noise <https://en.wikipedia.org/wiki/Colors_of_noise>`_.
 As well as not necessarily meeting technical definitions, referring to aperiodic neural activity
-as noise in this way is also different however to how the terms 'signal' & 'noise' are typically
-used in neuroscience, in terms of referring to a signal of interest and unwanted or artifactual
+as noise in this way is also different to how the terms 'signal' & 'noise' are typically
+used in neuroscience, in terms of referring to a signal of interest as unwanted or artifactual
```

Comment:
I have a tendency to write run-on sentences. I think this one would actually be clearer if it were split up into two sentences. Can you edit to make it a period where to comma is, at the end of `used in neuroscience`. And then the next sentence can be something like "In neuroscience, the terms 'signal' and 'noise' are typically used to refer to data of interest and a unwanted or artifactual data, respectively." (feel free to edit / tweak a little).